presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,49 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Improve performance of queries that use decimal type. ({issue}`4886`)
+* Improve query performance by adding support for dynamic filtering and dynamic
+  partition pruning to semi-join relational operator. ({issue}`5017`)
+* Improve dynamic partition pruning and query performance by reducing latency of dynamic filters collection. ({issue}`4988`)
+* Improve join performance when cost-based optimizer has missing or inaccurate stats. ({issue}`5141`)
+* Enable connectors to wait for dynamic filters derived from replicated join before generating splits. ({issue}`4685`)
+* Fix query failure when lambda expression references a table column containing a dot. ({issue}`5087`)
+
+## Atop Connector Changes
+
+* Fix incorrect query results when query contains predicates on `start_time` or `end_time` column. ({issue}`5125`)
+
+## Hive Connector Changes
+
+* Allow specifying what happens if data is inserted into existing Hive partition. 
+  This can be done using `hive.insert-existing-partitions-behavior` config property. ({issue}`4999`)
+* Add property (`hive.dynamic-filtering-probe-blocking-timeout`) for delaying table scans
+  until dynamic partition pruning can be performed more efficiently. ({issue}`4991`)
```

Comment:
It feels to me that we are using dynamic partition pruning and dynamic filtering interchangeably without consequence. I think for changes which relate to both we should stick to dynamic filtering, as this is more general feature and only use dynamic partition pruning for specific cases.

Reply:
makes sense, let's stick to `dynamic filtering`